docs/source/data/data_catalog.md

```diff
@@ -356,7 +356,7 @@ You can [configure parameters](../kedro_project_setup/configuration.md#load-para
 
 ## Feeding in credentials
 
-Before instantiating the `DataCatalog` Kedro will first attempt to read the credentials from [the project configuration](../kedro_project_setup/configuration.md#aws-credentials). The resulting dictionary is then passed into `DataCatalog.from_config()` as the `credentials` argument.
+Before instantiating the `DataCatalog` Kedro will first attempt [to read the credentials from the project configuration](../kedro_project_setup/configuration.md#aws-credentials). The resulting dictionary is then passed into `DataCatalog.from_config()` as the `credentials` argument.
```

Comment:
Before instantiating the `DataCatalog`, Kedro will first attempt to read the [credentials from the project configuration](../kedro_project_setup/configuration.md#aws-credentials).